src/Elastica/Client.php

```diff
@@ -24,6 +24,8 @@
  */
 class Client extends BaseClient
 {
+    private $forbiddenHttpCodes = [403, 404, 400];
```

Comment:
We should make this configurable.

Reply:
Agreed, let me implement this and add some tests

Reply:
Please note that by intercepting `404` you break various `exists` functions in Elastica which send `HEAD` requests and check for `200` vs `404` responses.
* `Elastica\Type::exists()`
* `Elastica\Index::exists()`
* `Elastica\IndexTemplate::exists()`